carsus/io/cmfgen/cmfgen_.py

```diff
@@ -0,0 +1,199 @@
+import numpy as np
+import pandas as pd
+import warnings
+from carsus.io.base import BaseParser
+
+
+def find_row(fname, string1, string2='', how='both', num_row=False):
+    """  """
+    with open(fname, encoding='ISO-8859-1') as f:
+        n = 0
+        for line in f:        
+            n += 1
+            
+            if how == 'both':
+                if string1 in line and string2 in line:
+                    break
+            
+            if how == 'first':
+                if string1 in line and string2 not in line:
+                    break
+        
+        # In case there's no match
+        else:
+            n = None
+            line = None
+    
+    if num_row == True:
+        return n
+                    
+    return line
+
+
+to_float = lambda x: float(x.replace('D', 'E'))  # string to float, taking care of Fortran 'D' values
```

Comment:
PEP8 recommends to use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier.  You can turn the comment into a docstring.

Reply:
Done.